draft-ietf-gnap-core-protocol.md

```diff
@@ -3321,25 +3327,31 @@ The AS validates that the token presented is associated with the management
 URL, that the AS issued the token to the given client instance, and that
 the presented key is appropriate to the token.
 
-If the access token has expired, the AS SHOULD honor the rotation request to
-the token management URL since it is likely that the client instance is attempting to
+Note that in many cases, the access token will have expired for regular use. To facilitate
+token rotation, the AS SHOULD honor the rotation request of the expired access token
+since it is likely that the client instance is attempting to
 refresh the expired token. To support this, the AS MAY apply different lifetimes for
 the use of the token in management vs. its use at an RS. An AS MUST NOT
-honor a rotation request for an access token that has been revoked, either by
-the AS or by the client instance through the [token management URI](#revoke-access-token).
+honor a rotation request for an access token that has been revoked or otherwise disabled.
```

Comment:
what do we mean by disabled ?

Reply:
Generically, just "made unusable for any reason whatsoever". Revocation is the most concrete -- and permanent -- version of that.